docs/workflows/copy.rst

```diff
@@ -14,7 +14,9 @@ Like all Pulp repositories, you can use the ${repo_href}/modify/ endpoint to:
 
 * add or remove individual content units from a repository by HREF
 * roll back the content present in a repository to that of a previous version using 'base_version'
-* clone a repository version from another repo using 'base_version'
+* clone a repository version from another repo using 'base_version'. This operation will copy all
+  the content from 'base_version' into the repository by replacing the latest available. All content
+  that was in the repository previously wild be removed.
```

Comment:
Typo "wild"

Reply:
I think I would actually replace this entire section with something like:

>* clone another repository version using 'base_version'.  This operation will create a new repository version in the current repository which is a copy of the one specified as the 'base_version', regardless of what content was previously present in the repository.  This can be combined with adding and removing content units in the same call, enabling work flows such as "create a repository version equivalent to X previous version, but with 2 packages added and one removed".

That might be a bit too much though.  What do you think?

Reply:
I will add your suggestion and reference to the Recipe section for more info